alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/YorcPaaSProvider.java

```diff
@@ -88,6 +89,23 @@
     private ProviderConfig providerConfiguration;
     private Map<String, String> a4cDeploymentIds = Maps.newHashMap();
 
+    /**
+     * Keep in mind the pass (yorc) deploymentIds that have no
+     * correspondent deploymentId in Alien. This can arrive for multiple reasons:
+     * - deployment was created with yorc's CLI
+     * - deployment was created with another Alien instance
+     * - Alien was restared after runtime removed
+     */
+    private List<String> unknownDeploymentIds = new ArrayList<>();
+
+    protected boolean isUnknownDeploymentId(String paasId) {
+        return unknownDeploymentIds.contains(paasId);
+    }
+
+    protected void setUnknownDeploymentId(String paasId) {
```

Comment:
I would have called this method ``addUnknownDeploymentId`` or something like that o make it understandable that it is actually a list of ids.